src/animation.cpp

```diff
@@ -97,6 +97,14 @@ tripoint relative_view_pos( player const &u, tripoint const &p ) noexcept
     return relative_view_pos( u, p.x, p.y, p.z );
 }
 
+// Convert p to screen position relative to the current terrain view
+static tripoint relative_view_pos( game const &g, tripoint const &p ) noexcept
```

Comment:
We usually write `const type` not `type const`.

Reply:
The other reload of this function uses this format, see the above code.